### ISSUE TITLE
Error: Cannot find module './dist/cpexcel.js'

### ISSUE THREAD

**Issue by texiontech** [gitbrent/xlsx-js-style#6]
Node version :  node:16-alpine3.15
Version : xlsx-js-style  : "^1.10"

backend      | Error: Cannot find module './dist/cpexcel.js'
backend      | Require stack:
backend      | - /usr/src/app/node_modules/xlsx-js-style/dist/xlsx.min.js

on version 1.0 it's work

**Comment by duketide** [comment]
I am experiencing a similar issue when attempting an ES6 import.

> ERROR in ./node_modules/xlsx-js-style/dist/xlsx.min.js 9:120-148
> Module not found: Error: Can't resolve './dist/cpexcel.js' in '<path>\node_modules\xlsx-js-style\dist'

**Comment by akanzari** [comment]
I have the same problem today 
 ERROR in ./node_modules/xlsx-js-style/dist/xlsx.min.js 
Module not found: Error: Can't resolve './dist/cpexcel.js' in '/webapp/node_modules/xlsx-js-style/dist'

**Comment by mephasor** [comment]
Same here, I believe the change in line 19 of [this commit](https://github.com/gitbrent/xlsx-js-style/commit/dada46ca4e80f3b5f30ef4d2380b79d8761724c2) might be a typo. 

reverting to version 1.0.0 of the npm package solves it for now

**Comment by gitbrent** [comment]
Thanks @texiontech et. al. - sorry about that, the 1.1.0 release completely goofed up dependencies.

This issue is fixed in [Release v1.2.0](https://github.com/gitbrent/xlsx-js-style/releases/tag/v1.2.0).

There is also a new node demo for package testing going forward.

```bash
[iMac 20:51:53] ~/GitHub/xlsx-js-style/demos/node 
=> node demo.js
--------------------==~==~==~==[ STARTING DEMO... ]==~==~==~==--------------------
`XLSX.version` ......... = 0.18.5
`XLSX.style_version` ... = 1.2.0
--------------------==~==~==~==[ ...DEMO COMPLETE ]==~==~==~==--------------------
```
